client/landing/stepper/declarative-flow/internals/steps-repository/blogger-intent/style.scss

```diff
@@ -66,10 +70,12 @@
 			align-items: center;
 			color: var(--color-neutral-100);
 			margin-top: 24px;
-
+			margin-right: 40px;
```

Comment:
Avoid issues with some mobile screen sizes:

<img width="448" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/19283ff7-a433-4ef3-a4fa-2a6f11ff04ea">

Reply:
I might try 100% width on the row text for mobile breakpoints. Just adding a small margin might not work on some locale / translations.

Reply:
oh this is in your commit, lets not over think it, let's ship this as is.